data/json/items/gunmod/underbarrel.json

```diff
@@ -394,6 +398,7 @@
     "location": "underbarrel",
     "mod_targets": [ "pistol", "smg", "crossbow" ],
     "dispersion_modifier": 15,
+    "mode_modifier": [ [ "REACH", "bayonet", 2, [ "MELEE" ] ] ],
```

Comment:
...however the absence of a reach attack for pistol bayonet could have been intentional?

We could consider reducing the range to 1 or possibly dropping pistol bayonets altogether?